spec/features/shelters/pets_index_spec.rb

```diff
@@ -6,8 +6,10 @@
     @shelter_2 = Shelter.create(name: "Boulder shelter", city: "Boulder, CO", foster_program: false, rank: 9)
     @pet_1 = Pet.create(adoptable: true, age: 1, breed: "sphynx", name: "Bare-y Manilow", shelter_id: @shelter.id)
     @pet_2 = Pet.create(adoptable: true, age: 3, breed: "doberman", name: "Lobster", shelter_id: @shelter.id)
-    @pet_3 = Pet.create(adoptable: true, age: 1, breed: "domestic shorthair", name: "Sylvester", shelter_id: @shelter_2.id)
-    @pet_4 = Pet.create(adoptable: true, age: 1, breed: "orange tabby shorthair", name: "Lasagna", shelter_id: @shelter.id)
+    @pet_3 = Pet.create(adoptable: true, age: 1, breed: "domestic shorthair", name: "Sylvester",
```

Comment:
If you are going to break up attributes for one type of object you should do the same to all and you should break up all attributes not just cut them in half.

Reply:
Sorry, I didnt see Rubocop do this